contrib/pkg/createcluster/create.go

```diff
@@ -263,21 +281,21 @@ func (o *Options) GenerateObjects() ([]runtime.Object, error) {
 	if imageSet != nil {
 		result = append(result, imageSet)
 	}
+
+	// Adding pull secret name to cluster deployment if pull secret exists
+	if pullSecret != nil {
+		cd.Spec.PullSecret = &corev1.LocalObjectReference{
+			Name: fmt.Sprintf("%s-pull-secret", cd.Name),
+		}
+	}
```

Comment:
I am not sure if this is an acceptable, however I am open to suggestion.

Reply:
seems fine to me. if you want to make it a little less magical, i would store the "pull secret name" template as some const pullSecretNameTemplate = "%s-pull-secret", and just use it in your Sprintf() (not sure how much practical difference it makes...)

Reply:
Yeah, it will be more and less the same thing.